frontend/src/citizen-frontend/children/sections/service-need-and-daily-service-time/NewServiceApplicationPage.tsx

```diff
@@ -100,23 +101,40 @@ export default React.memo(function NewServiceApplicationPage() {
             <FixedSpaceColumn spacing="m">
               <FixedSpaceColumn spacing="s">
                 <Label>{i18n.children.serviceApplication.startDate} *</Label>
+                <div>{i18n.children.serviceApplication.startDateInfo}</div>
                 <DatePickerF
                   bind={startDate}
                   locale={lang}
                   hideErrorsBeforeTouched
                   info={startDate.inputInfo()}
                   data-qa="start-date"
                 />
+                {startDate.isValid() && startDate.value().getDate() !== 1 && (
```

Comment:
Tämän voisi wrapata tietyissä tapauksissa ilmestyvän elementtin voisi wrapata elementtiin jossa on `aria-live="true"`, niin ruudunlukijan käyttäjille lausutaan ilmestynyt info:
```suggestion
                <div aria-live="polite">{startDate.isValid() && startDate.value().getDate() !== 1 && (
```

Reply:
pitäiskö tuon olla AlertBoxin feature jonka sais propsilla päälle?

Reply:
näin? https://github.com/espoon-voltti/evaka/pull/5769/commits/2fdbff17cdd625d66441ecdb242c84fa34e97203

Reply:
ei toiminutkaan noin, tehty alkuperäisen ehdotuksen mukaan